js/pages/backoffice/toolbox.js

```diff
@@ -394,7 +395,7 @@ CombodoModal.OpenConfirmationModal = function(oOptions, aData) {
 				}
 			},
 			{
-				text: Dict.S('UI:Button:Ok'),
+				text: oOptions.confirm_button_label != null ? oOptions.confirm_button_label : Dict.S('UI:Button:Confirm'),
```

Comment:
```suggestion
				text: oOptions.confirm_button_label ?? Dict.S('UI:Button:Confirm'),
```

Can we simplify it like this? I think so.